content/applications/general/integrations/translate.rst

```diff
@@ -0,0 +1,122 @@
+================
+Google Translate
+================
+
+*Google Translate* can be used to translate user generated text in the Odoo chatter. To translate a
+user's text from another language, click the :guilabel:`...(three dot)` icon menu to the right of
+the chatter. Then, select :guilabel:`Translate`. The content translates to the *language* set on the
+user's preferences.
+
+.. seealso::
+   :ref:`language/change-user-language`
+
+.. image:: translate/google-translate.png
+   :align: center
+   :alt: Google Translate present in an Odoo database's chatter.
+
+Integration is simple with some configuration on *Google's Developer Console*, along with copying
+and pasting an *API key* into Odoo.
```

Comment:
This sentence is essentially saying the same thing as the opening paragraph in the **Google API console** section, I think you can just lead with that, but it is up to you!

```suggestion
```